velox/functions/prestosql/tests/ArraySortTest.cpp

```diff
@@ -583,6 +542,21 @@ TEST_F(ArraySortTest, lambda) {
       "(x, y) -> if(length(x) < length(y), 1, if(length(x) = length(y), 0, -1))");
 }
 
+TEST_F(ArraySortTest, failOnMapTypeSort) {
+  auto data = makeRowVector({makeAllNullMapVector(10, INTEGER(), VARCHAR())});
+  auto testFail = [&](const std::string& name, const std::string& lambdaExpr) {
+    VELOX_ASSERT_THROW(
+        evaluate(fmt::format("{}(c0, {})", name, lambdaExpr), data),
+        "(1 vs. 0)");
+    VELOX_ASSERT_THROW(
+        evaluate(fmt::format("{}(c0)", name), data),
+        "Scalar function signature is not supported");
+  };
+
+  testFail("array_sort", "x -> x");
```

Comment:
nit: It is confusing that this call passes a lambda. This makes it look like we are only testing lambda version of the function. Since the lambda is the same in all calls, consider removing lambda argument and hard-coding it inside testFail.

Reply:
Fixed.